lib/matplotlib/tests/test_patches.py

```diff
@@ -503,7 +503,56 @@ def test_fancyarrow_units():
 
 @image_comparison(["large_arc.svg"], style="mpl20")
 def test_large_arc():
-    ax = plt.figure().add_subplot()
-    ax.set_axis_off()
-    # A large arc that crosses the axes view limits.
-    ax.add_patch(mpatches.Arc((-100, 0), 201, 201))
+    fig, (ax1, ax2) = plt.subplots(1, 2)
+    x = 210
+    y = -2115
+    diameter = 4261
+    for ax in [ax1, ax2]:
+        a = mpatches.Arc((x, y), diameter, diameter, lw=2, color='k')
+        ax.add_patch(a)
+        ax.set_axis_off()
+        ax.set_aspect('equal')
+    # force the high accuracy case
+    ax1.set_xlim(7, 8)
+    ax1.set_ylim(5, 6)
+
+    # force the low accuracy case
+    ax2.set_xlim(-25000, 18000)
+    ax2.set_ylim(-20000, 6600)
```

Comment:
Is this still the right trigger?

Reply:
The second axis is trying to hit the "small" path which this does.  These numbers could be made much smaller now that we have fixed the quadratic growth issue.